packages/terra-base/src/baseStyles.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * The following code is used to ensure global styles are loaded only once in an app
+ * that consumes terra components.
+ */
+if (!global.baseStylesLoaded) {
```

Comment:
Looks like webpack will automatically dedupe this and the webpack extract text plugin will only include this once. I removed the if condition: https://github.com/cerner/terra-core/pull/251/commits/6b700b9a8487dc9a937a2451ccdabc75df6714ed